### ISSUE TITLE
 error: The "#jsx-fragment" element is not allowed in the "div" element in this context (permitted-contents)

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#1767]
- Markuplint version: `4.x.x`
- Parser lang: JSX
- Node.js version: `22.x.x`
- OS: macOS

## Code Example or Playground URL

```jsx
<div>
  <>
    ...
  </>
</div>
```

https://playground.markuplint.dev/#N4Igxg9gdgZglgcxALhMAOlABF9ICmAHgC75QAmAznslgNqY454C2AhgE4DWArgA4AbOFGLIO+SCxZly+cnkZYAugBpFePp0r4ONLBmxMF6EwDoAVpUIASPXgAC7bvyEiA9JcIBaTR226QRQBfNUM8Sj4JahR9RWZAk3QLK1sYhydeQWFiN3E2MGIvCIkFQyDMIJAVcAhZADE4AXwAFQBPSJQQZMIqmtlOgB5yOAA3AD5FAYnDHFM5ybdpgbdh8ZAgoA

## The current behavior

>  error: The "#jsx-fragment" element is not allowed in the "div" element in this context (permitted-contents)

## The expected behavior

No error, and it expects to evaluate transparently.
